examples/benchmark_stem_image.py

```diff
@@ -0,0 +1,23 @@
+import glob
+from stempy import io, image
+import time
+
+# Beware of disk caching when using this benchmark
+num_runs = 10
+times = []
+for i in range(num_runs):
+    start = time.time()
+
+    files = []
+    for f in glob.glob('/data/4dstem/smallScanningDiffraction/data*.dat'):
```

Comment:
Can we using [click](http://click.palletsprojects.com/en/7.x/) to allow the path to be passed in?

Reply:
Sure!